docker/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:latest
```

Comment:
I would prefer using a fixed LTS version of node, i.e., `node:14.5`  instead of latest, as it can cause unexpected errors when a new version of node rolls out.

Reply:
Completed all modifications that were needed.